test_unstructured/partition/test_text_type.py

```diff
@@ -211,12 +211,14 @@ def test_contains_english_word(text, expected, monkeypatch):
 @pytest.mark.parametrize(
     ("text", "expected"),
     [
-        ("Intellectual Property in the United States", True),
+        ("Intellectual Property in the United States", False),
         ("Intellectual property helps incentivize innovation.", False),
         ("THIS IS ALL CAPS. BUT IT IS TWO SENTENCES.", False),
         ("LOOK AT THIS IT IS CAPS BUT NOT A TITLE.", True),
         ("This Has All Caps. It's Weird But Two Sentences", False),
         ("The Business Report is expected within 6 hours of closing", False),
+        ("United States' Intellectual Property", True),
+        ("United States' intellectual property", False),
```

Comment:
🎆 but do have to say this took me a second to think through "yes, this makes sense" 😅